compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -2743,6 +2743,12 @@ impl<'tcx> TypeErrCtxtExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
             }
             ObligationCauseCode::BindingObligation(item_def_id, span)
             | ObligationCauseCode::ExprBindingObligation(item_def_id, span, ..) => {
+                if self.tcx.is_diagnostic_item(sym::Send, item_def_id)
```

Comment:
why isn't this commented? seems like something that could use a justification

Reply:
I'll add a comment.